tutorials/algorithms/03_vqe_simulation_with_noise.ipynb

```diff
@@ -186,158 +229,117 @@
-      "needs_background": "light"
-     },
-     "output_type": "display_data"
-    }
-   ],
+   "outputs": [],
    "source": [
-    "if counts1 or values1:\n",
-    "    pylab.rcParams['figure.figsize'] = (12, 4)\n",
-    "    pylab.plot(counts1, values1)\n",
-    "    pylab.xlabel('Eval count')\n",
-    "    pylab.ylabel('Energy')\n",
-    "    pylab.title('Convergence with noise')"
+    "noisy_estimator = AerEstimator(\n",
+    "    backend_options={\n",
+    "        \"method\": \"density_matrix\",\n",
+    "        \"coupling_map\": coupling_map,\n",
+    "        \"noise_model\": noise_model,\n",
+    "    },\n",
+    "    run_options={\"seed\": seed},\n",
+    "    transpile_options={\"seed_transpiler\": seed},\n",
+    ")"
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": 10,
    "metadata": {},
+   "outputs": [],
    "source": [
-    "## Performance *with* noise and measurement error mitigation\n",
+    "# re-define callback\n",
+    "# note: Re-run this cell to restart lists before training\n",
+    "counts = []\n",
+    "values = []\n",
     "\n",
-    "Now we will add method for measurement error mitigation, which increases the fidelity of measurement. Here we choose `CompleteMeasFitter` to mitigate the measurement error. The calibration matrix will be auto-refresh every 30 minute (default value).\n",
     "\n",
-    "Note: simulation with noise will take longer than without noise."
+    "def store_intermediate_result1(eval_count, parameters, mean, std):\n",
+    "    counts.append(eval_count)\n",
+    "    values.append(mean)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
-   "metadata": {
-    "scrolled": true
-   },
+   "execution_count": 11,
+   "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "VQE on Aer qasm simulator (with noise and measurement error mitigation): -1.86029\n",
-      "Delta from reference energy value is -0.00301\n"
+      "VQE on Aer qasm simulator (with noise): -1.74645\n",
+      "Delta from reference energy value is 0.11083\n"
      ]
     }
    ],
    "source": [
-    "from qiskit.utils.mitigation import CompleteMeasFitter\n",
-    "\n",
-    "counts2 = []\n",
-    "values2 = []\n",
-    "if noise_model is not None:\n",
-    "    algorithm_globals.random_seed = seed\n",
-    "    qi = QuantumInstance(backend=backend, seed_simulator=seed, seed_transpiler=seed,\n",
-    "                         coupling_map=coupling_map, noise_model=noise_model, \n",
-    "                         measurement_error_mitigation_cls=CompleteMeasFitter, \n",
-    "                         cals_matrix_refresh_period=30)\n",
-    "\n",
-    "    def store_intermediate_result2(eval_count, parameters, mean, std):\n",
-    "        counts2.append(eval_count)\n",
-    "        values2.append(mean)\n",
+    "vqe = VQE(noisy_estimator, ansatz, optimizer=spsa, callback=store_intermediate_result)\n",
```

Comment:
Now we could just re-use the prior VQE instance right, setting the estimator and possibly the new callback if we want to retain the former results (rather that say just clearing its lists and reusing that same callback). Might be more interesting as a different way to do things rather than seeing VQE's built out from scratch each time?

Reply:
Good idea.